Bottom: af5df99376059d35e2cb5865e3f18a7223bbe4df
Top:    0bc39c2212dd2dbee9f5ff7c2d5c38d49102352d
Author: Pavel Roskin <proski@gnu.org>
Date:   2013-11-04 12:03:02 -0500

Refresh of reconfig

---

diff --git a/farm-config32 b/farm-config32
index 71ac62d..56ed509 100755
--- a/farm-config32
+++ b/farm-config32
@@ -20,8 +20,8 @@ for srctree in $farm/*; do
 	cd $srctree
 	$make HOSTCC="$ucc $ucflags" KBUILD_OUTPUT=$bintree \
 		CONFIG_WIRELESS_EXT=y CONFIG_NET_RADIO=y CONFIG_IPW2100=y \
-		CONFIG_CRYPTO=y ARCH=$arch defconfig >Log-config1-out \
-		2>Log-config1-err
+		CONFIG_CRYPTO=y ARCH=$arch defconfig >$bintree/Log-config1-out \
+		2>$bintree/Log-config1-err
 	cd $bintree
 	sed -i -e 's/# CONFIG_NET_RADIO is not set/CONFIG_NET_RADIO=y/' .config
 	sed -i -e 's/# CONFIG_WIRELESS_EXT is not set/CONFIG_WIRELESS_EXT=y/' .config
diff --git a/farm-config64 b/farm-config64
index a19fb9b..4189d8a 100755
--- a/farm-config64
+++ b/farm-config64
@@ -20,8 +20,8 @@ for srctree in $farm/*; do
 	cd $srctree
 	$make HOSTCC="$ucc $ucflags" KBUILD_OUTPUT=$bintree \
 		CONFIG_WIRELESS_EXT=y CONFIG_NET_RADIO=y CONFIG_IPW2100=y \
-		CONFIG_CRYPTO=y ARCH=$arch defconfig >Log-config1-out \
-		2>Log-config1-err
+		CONFIG_CRYPTO=y ARCH=$arch defconfig >$bintree/Log-config1-out \
+		2>$bintree/Log-config1-err
 	cd $bintree
 	sed -i -e 's/# CONFIG_NET_RADIO is not set/CONFIG_NET_RADIO=y/' .config
 	sed -i -e 's/# CONFIG_WIRELESS_EXT is not set/CONFIG_WIRELESS_EXT=y/' .config
